content/en/docs/quick-start/deploy-bookinfo-to-k8s.md

```diff
@@ -1,8 +1,221 @@
 ---
-title: "Deploy a Bookinfo App"
+title: "Deploy Bookinfo and Manage Traffic"
 keywords: 'kubesphere, kubernetes, docker, multi-tenant'
 description: 'Deploy a Bookinfo App'
 
-linkTitle: "Deploy a Bookinfo App"
+linkTitle: "Deploy Bookinfo and Manage Traffic"
 weight: 3040
 ---
+
+[Istio](https://istio.io/), as an open-source service mesh solution, provides powerful features of traffic management for microservices. Here is the introduction of traffic management from the official website of [Istio](https://istio.io/latest/docs/concepts/traffic-management/):
+
+*Istio’s traffic routing rules let you easily control the flow of traffic and API calls between services. Istio simplifies configuration of service-level properties like circuit breakers, timeouts, and retries, and makes it easy to set up important tasks like A/B testing, canary rollouts, and staged rollouts with percentage-based traffic splits. It also provides out-of-box failure recovery features that help make your application more robust against failures of dependent services or the network.*
+
+KubeSphere provides three kinds of grayscale strategies based on Istio, including blue-green deployment, canary release and traffic mirroring.
+
+Among others, a canary release represents an effective software development strategy in which a new version is deployed for testing with the base version preserved in the production environment. This strategy will bring part of the traffic to the new version being tested and the production release takes up the rest.
+
+## Objective
+
+In this tutorial, you will learn how to deploy a sample application Bookinfo composed of four separate microservices and use the traffic management feature of KubeSphere to publish a new version.
+
+## Prerequisites
+
+- You need to enable [KubeSphere Service Mesh](../../pluggable-components/service-mesh/).
+
+- You need to finish all tasks in [Create Workspace, Project, Account and Role](../create-workspace-and-project/).
+
+- You need to enable **Application Governance**. To do so, follow the steps below:
+
+  Log in the console as `project-admin` and go to your project. Navigate to **Advanced Settings** under **Project Settings**, click **Edit**, and select **Edit Gateway**. In the dialog that appears, flip on the toggle switch next to **Application Governance**.
+
+![edit-gateway](https://ap3.qingstor.com/kubesphere-website/docs/20200908145220.png)
+
+![switch-application-governance](https://ap3.qingstor.com/kubesphere-website/docs/20200908150358.png)
```

Comment:
You would better to add a notice after enabling Application Governance, it will cause the routes (ingress) inaccessible in the namespace and requires the corresponding configuration as its highlight.

![image](https://user-images.githubusercontent.com/40452856/92472185-b8f75380-f20b-11ea-8b54-5220797b2f63.png)